crates/interpreter/src/gas/calc.rs

```diff
@@ -193,50 +193,56 @@ pub fn sstore_cost<SPEC: Spec>(
     gas: u64,
     is_cold: bool,
 ) -> Option<u64> {
```

Comment:
This whole function could be seen as.
```
if SPEC::enabled(BERLIN) {

} else if  SPEC::enabled(ISTANBUL) {

} else {

}
```

Where berlin path would call instanbul function as they have similar functionality.

Reply:
Resolved, I've left the 1706 check for Istanbul outside the main if-else condition statement though